pkg/client/secrets.go

```diff
@@ -64,3 +133,20 @@ func (c *HTTPClient) ListSecrets(ctx context.Context, storeName string) ([]strin
 
 	return ids, nil
 }
+
+func (c *HTTPClient) ListDeletedSecrets(ctx context.Context, storeName string) ([]string, error) {
+	var ids []string
+	reqURL := fmt.Sprintf("%s/%s/deleted", withURLStore(c.config.URL, storeName), secretsPath)
```

Comment:
`GetDeleted` and `ListDeleted` are just Get and List but with a query param. This is the example for Eth1 in the handler:
```
getDeleted := request.URL.Query().Get("deleted")
	var eth1Acc *entities.ETH1Account
	if getDeleted == "" {
		eth1Acc, err = eth1Store.Get(ctx, getAddress(request))
	} else {
		eth1Acc, err = eth1Store.GetDeleted(ctx, getAddress(request))
	}
```

Reply:
I will apply same pattern here